### ISSUE TITLE
Client-side image resizing stuck with endless spinner 

### ISSUE THREAD

**Issue by jacmaes** [processwire/processwire-issues#318]
### Short description of the issue
As mentioned in the [comments following the forum post announcement](https://processwire.com/talk/topic/16412-pw-3063-client-side-image-resize-support/) of client-side image resizing, this new function seems to work erratically for some of us. I haven't been able to pinpoint the issue exactly, but it seems to occur with large images, i.e. larger than 3000 pixels.

### Expected behavior

The image get resized client-side.

### Actual behavior

The spinner icon spins and spins and spins...

### Steps to reproduce the issue

1. Upload a large JPEG
2. Watch how the spinner keeps moving but no upload happens. 

### Setup/Environment

* ProcessWire version: Latest dev
* (Optional) PHP version: 7.1
* (Optional) Any 3rd party modules that are installed and could be related to the issue: Croppable Image 3.

**Comment by teppokoivula** [comment]
@jacmaes: though this may be unrelated, it would be interesting to see if there are any JavaScript errors etc. in this case. If you're able to reproduce this issues, could you check for those from your browser's console tab for those?

**Comment by ryancramerdesign** [comment]
I'm not seeing the issue here, but was wondering if you could try the following:

1. Try without Croppable Image 3, any difference?
2. Any JS errors, per @teppokoivula message?
3. Since this is client side, browser matters–what browser are you using (if not Chrome)?

**Comment by jacmaes** [comment]
It's been working fine recently, so I can't really answer you until I'm able to reproduce it. There might have been a bug in a recent Chrome release that's been fixed (I'm using the stable releases). The only other thing that's changed ––although I don't think it can affect this–– is that I upgraded the RAM in my VPS. I also disabled my ad blocker (uBlock Origin) that might have blocked or interfered with the JS.
